test/commands/execute.spec.ts

```diff
@@ -0,0 +1,68 @@
+import dotenv from "dotenv";
+dotenv.config();
+
+// Import Node.js Dependencies
+import { fileURLToPath } from "node:url";
+import path from "node:path";
+import fs from "node:fs/promises";
+import { afterEach, describe, it } from "node:test";
+import assert from "node:assert";
+
+// Import Third-party Dependencies
+import stripAnsi from "strip-ansi";
+
+// Import Internal Dependencies
+import { filterProcessStdout } from "../helpers/reportCommandRunner.js";
+import * as CONSTANTS from "../../src/constants.js";
+
+// CONSTANTS
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const processDir = path.join(__dirname, "../..");

```

Comment:
`process.cwd()` should work, no ?
```suggestion
const processDir = process.cwd();
```
You also miss the `kConst` convention:
```suggestion
const kProcessDir = process.cwd();
```

Reply:
Yes but it will be replaced by var/temp as suggested by Thomas